### PR TITLE
Added ability to Load & Save Network Settings from Local Storage & Realistic V2 Theme. Updated Dependencies to Latest Version & More!

### PR DESCRIPTION
**Pull request by TheBaconCat**
Here is my pull request of changes. I still need to finish some Gifs on my end. So I would say this is still a WIP Version

### CODE REVIEW

File: LICENSE
Comment (LapsTimeOFF) [inline LICENSE]: DigiFlag Team -> F1MV_DigiFlag Team (@LapsTimeOFF, @TheBaconCat, @JJWatMyself, @f1multiviewer)

**Comment by JJWatMyself** [comment]
So cool to see the collaboration on this.  Thank you team for all the amazing work!

File: LICENSE
Comment (LapsTimeOFF) [reply LICENSE]: Now updated -> commit 1296ac5

**Comment by LapsTimeOFF** [comment]
So, I will soon push to GitHub :
- Rain

**Review by LapsTimeOFF — COMMENTED** [review]
Very good work. I will give my Approve feedback when I will add ~~the telemetry and~~ the rain

**Comment by LapsTimeOFF** [comment]
There is a little graphic issue

https://user-images.githubusercontent.com/39061148/196250947-8a19a0ca-e3d0-4bfc-a289-9069a22bdc94.mov

**Comment by LapsTimeOFF** [comment]
Btw I've some gif for the new realistic mode, I will push them ASAP

File: index.js
Comment (LapsTimeOFF) [inline index.js]: Why * 1 ?

File: filesConfiguration.json
Comment (LapsTimeOFF) [inline filesConfiguration.json:1]: Rain is missing everywhere, adding all the references

File: index.js
Comment (TheBaconCat) [reply index.js]: I never added that lol. That has been there since as far as I can tell v1.1.0-stable [ selectTheme(e.target.id * 1)](https://github.com/LapsTimeOFF/DigiFlag_F1MV/blob/a8aa4d5d8a9fca1919a6a01cc675557fcc0ab31c/index.js#L177)

Comment (LapsTimeOFF) [reply index.js]: Okay, idk why I've put that

**Review by LapsTimeOFF — COMMENTED** [review]
Very good, still waiting for the "final" version before I approve

**Review by LapsTimeOFF — COMMENTED** [review]
Okayy, very good, no more scaling issue. Let us know when you're on the final version !

**Comment by TheBaconCat** [comment]
Ready for Merge! Finally!

**Review by LapsTimeOFF — APPROVED** [review]
Okay, the code is very clean. No big issue found, you're getting my approval ! Still waiting for the @JJWatMyself review though

**Comment by TheBaconCat** [comment]
Going to address some corrections as mentioned by @JJWatMyself 

1. The animations / realism for these GIF files are inconsistent with the other images:

realistic_v2_169\dyellow.gif
realistic_v2_169\sc.gif

2. I also think that 'Realistic v2 1:1' should be renamed to 'Square v2' as it's not truly realistic.

**Review by TheBaconCat — COMMENTED** [review]
@JJWatMyself How about these updated Gifs

**Review by LapsTimeOFF — COMMENTED** [review]
Perfect !

**Review by TheBaconCat — COMMENTED** [review]
Here is where I found the [Digital Flag Types](https://www.emmotorsport.com/solutions/circuits)

**Review by LapsTimeOFF — APPROVED** [review]
Perfect !

**Pull request merged**
